StripeFinancialConnections/StripeFinancialConnections/Source/Web/FinancialConnectionsWebFlowViewController.swift

```diff
@@ -84,7 +84,8 @@ final class FinancialConnectionsWebFlowViewController: UIViewController {
         return item
     }()
 
-    private let loadingView = LoadingView(frame: .zero)
+    // Use nil theme so the spinner view doesn't flash to the theme's color before launching the webview.
+    private let loadingView = LoadingView(frame: .zero, theme: nil)
```

Comment:
This fixes a small issue where the spinner would flash to the theme's color right before launching the web flow.

Before:

https://github.com/user-attachments/assets/ab4c55b3-ba22-4b11-aa5b-9331b1cc4cc2

After:

https://github.com/user-attachments/assets/cfebaa54-2d24-49f9-b9fc-0e6a19203a1b